velox/docs/functions/spark/math.rst

```diff
@@ -18,6 +18,14 @@ Mathematical Functions
 
     Returns inverse hyperbolic sine of ``x``.
 
+.. spark:function:: atan2(y, x) -> double
+
+    Returns the arc tangent of ``y / x``. For compatibility with Spark, returns 0 for the following corner cases:
+    * if y and x are both 0.
```

Comment:
It might be easier to read if these were written as functions calls:

```
* atan2(0.0, 0.0)
* atan2(-0.0, -0.0)
* atan2(-0.0, 0.0)
* atan2(0.0, -0.0)
```

Also, do you think readers may wonder what is -0.0 and how is it different from 0.0? Any thing we can add to help these readers?